.travis.yml

```diff
@@ -87,14 +89,10 @@ before_script:
     - sed -i "s/yourusernamehere/root/" wp-tests-config.php
     - sed -i "s/yourpasswordhere//" wp-tests-config.php
     - cd "/tmp/wordpress/src/wp-content/plugins/$PLUGIN_SLUG"
-    - gem install sass
-    - gem install compass
-    - nvm install 5
-    - npm install -g npm@'3.8.9'
-    - npm install -g gulp-cli
-    - npm install
+    - ./tests/load-travis.sh
 
-script: gulp $WP_TRAVISCI
+script:
+    - $WP_TRAVISCI
```

Comment:
fwiw it looks like multisite testing had been commented out anyway... https://github.com/Automattic/jetpack/blob/branch-4.0/Gruntfile.js#L141-L144